src/App.tsx

```diff
@@ -15,48 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortedBy: SortType;
+  isReversed: boolean;
+};
+
+const getGoods = (
+  goods: string[],
+  { sortedBy, isReversed }: ReorderOptions,
+) => {
+  const visibleGoods = Array.from(goods);
```

Comment:
Also you can do this way, more shorter
```suggestion
  const visibleGoods = [...goods];
```